cmd/cli/app/auth/offline_use.go

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/stacklok/minder/internal/config"
+	clientconfig "github.com/stacklok/minder/internal/config/client"
+	"github.com/stacklok/minder/internal/util"
+	"github.com/stacklok/minder/internal/util/cli"
+)
+
+// offlineTokenUseCmd represents the offline-token use command
+var offlineTokenUseCmd = &cobra.Command{
+	Use:   "use",
+	Short: "Use an offline token",
+	Long: `The minder auth offline-token use command project lets you install and use an offline token
+for the minder control plane.
+
+Offline tokens are used to authenticate to the minder control plane without
+requiring the user's presence. This is useful for long-running processes
+that need to authenticate to the control plane.`,
+
+	RunE: func(cmd *cobra.Command, _ []string) error {
+		clientConfig, err := config.ReadConfigFromViper[clientconfig.Config](viper.GetViper())
+		if err != nil {
+			return cli.MessageAndError("Unable to read config", err)
+		}
+
+		f := cmd.Flags().Lookup("file").Value.String()
```

Comment:
We should be able to use something like `fileFlag := viper.GetString("file")`

Reply:
I tried and it didn't set the default I needed. I could use some help in figuring out why

Reply:
That's strange, I did a quick look at the `profile` command as an example for the `provider` flag and we don't do anything different there. If you can't make it work we can go like that and figure this out later.

Reply:
Lol, that's exactly what I looked at as well... yeah, just couldn't figure it out

Reply:
Yeah, leave it like that. I recall that the flags should be bound to Viper for this to work but funny why it's working in one place and not on another.